exporter/debugexporter/exporter.go

```diff
@@ -60,6 +71,22 @@ func (s *debugExporter) pushTraces(_ context.Context, td ptrace.Traces) error {
 		return nil
 	}
 
+	for _, rs := range td.ResourceSpans().All() {
+		rs.Resource().Attributes().RemoveIf(func(k string, _ pcommon.Value) bool {
```

Comment:
In my mind, the `output::attributes` only affects the output of record-level attributes (log record attributes, metric data point attributes, span attributes). The output of resource or scope level attributes should be governed by separate options `output::resource` and `output::scope`.

Perhaps better to name the option `output::record::attributes`? What do you think?